modules/mobile/app/controllers/mobile/v0/facilities_info_controller.rb

```diff
@@ -5,8 +5,13 @@
 module Mobile
   module V0
     class FacilitiesInfoController < ApplicationController
+      SORT_METHODS = %w[home current alphabetical appointments].freeze
```

Comment:
I thought it would be better to define valid sort methods up front so that we can fail earlier in the process. Before, failure would happen after most of the logic was completed. Now, we don't bother doing anything unless the provided sort method is valid